lib/application.js

```diff
@@ -253,9 +253,13 @@ function respond(ctx) {
 
   // status body
   if (null == body) {
-    if (ctx.response._explicitNullBody) {
-      ctx.response.remove('Content-Type');
-      ctx.response.remove('Transfer-Encoding');
+    // The source code at response.js's line 146 to 148 has removed Content-Type & Transfer-Encoding when set null to body.
+    // if (ctx.response._explicitNullBody) {
+    //   ctx.response.remove('Content-Type');
+    //   ctx.response.remove('Transfer-Encoding');
+    //   return res.end();
+    // }
+    if (body === null) {
```

Comment:
I don't understand this comment at all. You're referencing line 146-148 and those are empty lines in response.js file - or specifically a closing bracket and a comment.

Reply:
when body is null and "Content-Type" & 'Transfer-Encoding' had be removed from response.js before. and then at next code .still res.end() when body is null. So this code is unnecessary